index.js

```diff
@@ -125,5 +136,11 @@ const getColor = (percentile) => {
 sense.Leds.lowLight = true
 sense.Leds.clear()
 sense.Leds.flipH()
-run()
 
+run();
+
+// If we're event driven, we only ever do something if an event drops on us
+if (eventDriven) {
+  // Initiate
+  frontInstance.registerEvents({ port: 80 }, run);
```

Comment:
The data sent with the event is a preview, which lets you know the source of the event, but you still have to go an request the data for a conversation/inbox for example, if you want the actual contents. So we could probably optimise this, if we just wanted to, but we might still have to get conversation data to see whether it's a comment/tag/message/etc., else you'd be alerted, for example, to someone adding a tag, which probably isn't what we want.

Here's the relevant part of the docs: http://dev.frontapp.com/#events